doozer/doozerlib/cli/scan_osh.py

```diff
@@ -1,29 +1,336 @@
+
+        return False
+
+    def create_update_ocpbugs_ticket(self, packages: dict):
+        """
+        Check if an OCPBUGS ticket already exists for a particular package. If it doesn't create one and update
+        the contents to point to the latest scan results
+        """
+        for brew_package_name in packages:
+            data = packages[brew_package_name]
+
+            distgit_name = self.get_distgit_name_from_brew_nvr(data["nvr"])
+
+            image_meta: ImageMetadata = self.runtime.image_map[distgit_name]
+
+            try:
+                if not image_meta.scanning.jira_integration.enabled:
+                    self.runtime.logger.info(f"Skipping OCPBUGS creation for package {data['package_name']} "
+                                             f"since disabled in image metadata")
+                    continue
+            except AttributeError:
+                # Since this will fail until we set it up in image metadata
+                self.runtime.logger.info(f"Skipping OCPBUGS creation for package {data['package_name']} "
+                                         f"since disabled in image metadata")
+                continue
+
+            # Returns project and component name
+            _, potential_component = image_meta.get_jira_info()
+
+            # Check if ticket exits
+            summary = f"{self.version} SAST scan issues for {brew_package_name}"
+
+            # Find if there is already a "OPEN" ticket for this component.
+            # A ticket is said to be "OPEN" if statusCategory <= 'Assigned'.
+            # If we detect a net-new security issue, we should open a new ticket instead of updating the one
+            # that is in the process of being fixed.
+            query = f"project={self.jira_project} AND ( summary ~ '{summary}' ) AND " \
+                    "statusCategory in ('New', 'Assigned')"
+            # Can use open_issues.pop().raw['fields'] to see all the fields for the JIRA issue, to test
+            open_issues = self.search_issues(query)
+            self.runtime.logger.info(f"Issues found with query '{query}': {open_issues}")
+
+            # Check if this is the first time that we are raising the ticket for this component
+            query = f"project={self.jira_project} AND ( summary ~ '{summary}' ) AND " \
+                    "statusCategory not in ('New', 'Assigned')"
+            closed_issues = self.search_issues(query)
+
+            previous_ticket_id = None
+            if not closed_issues and not open_issues:
+                # There are no tickets for this component, raise a ticket
+                self.skip_diff_check = True
+            else:
+                self.runtime.logger.info(f"Closed Issues: {closed_issues}")
+                previous_ticket_id = closed_issues.pop()  # Returned in LIFO (last-in, first-out) order.
+
+            description = "{*}{color:#FF0000}WARNING{color}{*}\n" + MESSAGE.format(package_name=brew_package_name,
```

Comment:
So OpenScanHub is working on adding new results to the `imp` list. Regardless, we need Devs to at least look at the results right. They may need to triage and figure out if its a vulnerability or not right. So keeping this would at least motivate them to look at them right. :thinking: